### ISSUE TITLE
Large Planning Times

### ISSUE THREAD

**Issue by marcelino-pensa** [unr-arl/mbplanner_ros#2]
Thanks for sharing this repository with the open-source community!
I just ran the simulated example. I started `roslaunch mbplanner mbplanner_m100_sim.launch
` and then clicked on `Start Planner` in the UI.
Things looked awesome other than it would take quite a bit of time to compute the trajectories, as shows below. Are there any compiling/execution optimizations that I am missing, or is this expected? Are there GPU optimizations (I am running on a machine without an external GPU, if that matters)?

```
         Build tree     : 34.130 (s)
         Compute gain   : 35.360 (s)
         Evaluate graph : 6.000 (s)
         Total          : 75.490 (s)
```

**Comment by tungdanganh** [comment]
Hi @marcelino-pensa ,
Did you compile with Release option: **-DMAKE_BUILD_TYPE=Release**

**Comment by tungdanganh** [comment]
Sorry: **-DCMAKE_BUILD_TYPE=Release**

**Comment by marcelino-pensa** [comment]
I thought I did, but I recompiled/deleted/cloned so many times because of [this issue](https://github.com/unr-arl/mbplanner_ws/issues/3), that I forgot to make it a release the last time I compiled.
Just made it into a release, and problem solved:
```
         Build tree     : 0.240 (s)
         Compute gain   : 0.300 (s)
         Evaluate graph : 0.030 (s)
         Total          : 0.570 (s)
```

It's amazing that it runs so much faster just by making the `Release` optimization! Any ideas why is this the case?
